### ISSUE TITLE
TypeScript: missing exports for StrictBreadcrumbProps and StrictSelectProps

### ISSUE THREAD

**Issue by kangaechigai** [Semantic-Org/Semantic-UI-React#4235]
### Steps
```ts
import { StrictBreadcrumbProps, StrictSelectProps } from 'semantic-ui-react';
```

### Expected Result
`StrictBreadcrumbProps` and `StrictSelectProps` interfaces are available.

### Actual Result
`Module '"semantic-ui-react"' has no exported member 'StrictBreadcrumbProps'. ts(2305)`

`'"semantic-ui-react"' has no exported member named 'StrictSelectProps'.
Did you mean 'StrictSegmentProps'? ts(2724)`

### Workaround:
```
import { StrictBreadcrumbProps } from "semantic-ui-react/src/collections/Breadcrumb";
import { StrictSelectProps } from "semantic-ui-react/src/addons/Select";
```
or
```
import { StrictBreadcrumbProps } from "semantic-ui-react/dist/commonjs/collections/Breadcrumb";
import { StrictSelectProps } from "semantic-ui-react/dist/commonjs/addons/Select";
```

### Version
2.0.3

### Testcase
https://codesandbox.io/s/semantic-ui-react-forked-0x9y5?file=/src/strictProps.tsx

**Comment by layershifter** [comment]
@kangaechigai thanks for reporting hand, will be fixed by #4275.

**Comment by layershifter** [comment]
Was fixed in [`semantic-ui-react@2.0.4`](https://github.com/Semantic-Org/Semantic-UI-React/releases/tag/v2.0.4).
